content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,222 @@
+* Issuer URL: Your AD FS issuer URL, typically this is your AD FS name plus `/adfs`
+* Client ID: the Client Identifier that we noted down in the beginning of the previous step
+* Client Secret: the Shared Secret that we generated in the previous step
+* Default Teams: leave empty if you don't want users without any groups claims to be assigned to a default group. If you do, enter the desired default group name here. If you use this option, be careful with how much access you give to that group.
+* Scopes: set this to `openid` and `allatclaims`
+* Email: leave at the default of `email`
+* First Name: leave at the default of `given_name`
+* Last Name: leave at the default of `family_name`
+* Spectro Team: leave at the default of `groups`
+
+![Enable Palette OIDC SSO](palette-sso-with-adfs-images/20-configure_palette_oidc.png)
+
+
+When all the information has been entered, click the **Enable** button at the button to enable SSO. You should receive an `OIDC configured successfully` message.
+
+
+## 3 - Create Teams in Palette
+
+All that's left to do now is to create Team(s) in Palette for the groups claim(s) that were configured in AD FS, and give them the appropriate permissions. For this example, we will create the `Admins` team and give it `Tenant Admin` permissions. You can repeat this for any other team that you've configured group claims for.
+
+Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **Users & Teams** --> **Teams** tab. Click the **+ Create Team** button.
+
+![Create Palette Team](palette-sso-with-adfs-images/21-create_team.png)
+
+
+Specify `Admins` in the **Team name** field. We don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
+
+![Name Palette Team](palette-sso-with-adfs-images/22-name_team.png)
+
+
+You'll be returned back to the list of Teams. Click on the newly created `Admins` Team to open its details screen. To give this team administrative access to the entire tenant (and thus all Projects in it), we will give this Team the `Tenant Admin` role. Click on **Tenant Roles** --> **+ Add Tenant Role**:
+
+![Palette Tenant Roles](palette-sso-with-adfs-images/23-tenant_roles.png)
+
+
+Click on **Tenant Admin** to enable the role, this automatically enables all other roles with fewer permissions. Click **Confirm** to add the role:
+
+![Add Tenant Role](palette-sso-with-adfs-images/24-add_tenant_role.png)
+
+
+You will see the `Roles have been updated` message to confirm the change. Repeat this procedure for any other teams you need, ensuring they're given the appropriate permissions. Finally click the `X` next to **Team Details** in the top left corner to exit this screen.
+
+Congratulations! You have now successfully configured Palette SSO based on OIDC with Microsoft AD FS!
```

Comment:
```suggestion
You have now successfully configured Palette SSO based on OIDC with Microsoft AD FS!
```
We don't congratulate the reader because that implies this is a mountainous task, which is debatable but it's best practice to avoid this type of language in tutorials and how-tos.